pallets/nft/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+authors = ['Anmol']
+description = 'FRAME pallet template for defining custom runtime logic.'
+edition = '2018'
+license = 'Unlicense'
+name = 'pallet-nft'
+readme = 'README.md'
+repository = 'https://github.com/substrate-developer-hub/substrate-node-template/'
+version = '0.1.0'
```

Comment:
Let's keep the header consistent with the other `Cargo.toml` files (like the [kiities-pallet](https://github.com/anmolnetwork/anmol-node/blob/master/pallets/kitties/Cargo.toml#L2-L10))